corona-warn-app/cwa-app-android
# Notification about new Risk Status shows when App is open (EXPOSUREAPP-10764)

## Testing
- Use test menu to change risk level
- Notice that notification is triggered now in foreground
- To test the badge, it is a little bit tricky ,since you will land in home screen anyway after test menu

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10764
